psi4/driver/driver_findif.py

```diff
@@ -0,0 +1,798 @@
+from __future__ import division
+from psi4.driver.p4util.exceptions import ValidationError
+from psi4.driver.qcdb import molecule
+from psi4 import core
+import numpy as np
+#import scipy.linalg as la
+from psi4.driver.p4util import block_diagonal_array
```

Comment:
Go ahead and delete the unused import. And maybe move numpy to the top. Informally, I try to separate into three import blocks:-- std lib, external, internal.